temporal_sqlalchemy/bases.py

```diff
@@ -109,6 +115,10 @@ def record_history(self,
                 changes = attributes.get_history(clocked, prop.key)
 
             if changes.added:
+                if is_strict_mode:
+                    assert not is_vclock_unchanged, \
+                        'flush() has triggered for a changed temporalized property outside of a clock tick'
+
```

Comment:
but for posterity: this change isn't going to be permanent -- it's to help us squash bugs until we can do the clock tick correctly